src/set_theory/cardinal/ordinal.lean

```diff
@@ -1183,3 +1185,87 @@ by simp [←bit1_zero]
 end bit
 
 end cardinal
+
+section omega_1
+
+namespace ordinal
+
+open cardinal
+open_locale ordinal cardinal
+
+/--
+The first uncountable ordinal in type universe `u`.
+You can use the notation `ω₁`.
+-/
+noncomputable def omega_1 : ordinal.{u} := (aleph 1).ord
```

Comment:
Obviously `ω₁` is very important in the context of the Borel hierarchy, but I'd recommend just doing what we were already doing, which is defining local notation ``ω₁` := (aleph 1).ord.out.α`.

Reply:
I also felt that `ω₁` on its own was too particular. At first glance, I would take the following (somewhat disruptive) approach:

 - Define `omega'` as above but using `aleph'` instead.
 - Define `omega` as above, no primes, with its argument as a subscript (is this possible?).
 - Set `ω` as a local notation for the few files in which it appears, or stick to `ω₀`.
 
I believe that this `omega` function is really fundamental, transcending the Borel stuff (for one example involving cohomology, check the comments below Goblot's theorem in page 20 of [this paper](https://www.ams.org/journals/bull/1974-80-01/S0002-9904-1974-13343-4/S0002-9904-1974-13343-4.pdf), or [a more recent one](https://arxiv.org/abs/2203.00165)).

Anyway, as you said, just sticking to `(aleph x).ord` might be simpler (with a subscript notation like `ℵₓ.ord` it would be better, but I could not make it work). Concerning the use of `.out`, see below.

Reply:
I like this idea. We could also rename the current `ordinal.omega` to `ordinal.omega_0`, matching `cardinal.aleph_0` (even though as far as I know, nobody uses the notation `ω₀`, weird). This would be a large change though, so it might be best posting on Zulip first.

Reply:
Sound like a good plan, but I don't think this is possible:

> Define omega as above, no primes, with its argument as a subscript (is this possible?).

Reply:
>  (even though as far as I know, nobody uses the notation `ω₀`, weird)

Only in ancient books, I concur. But as an afterthought: perhaps the cost of library coherence is to have a coherent notation (!) :rofl:

Reply:
> This would be a large change though, so it might be best posting on Zulip first.

Should I post it? Also, since it looks large, if accepted, should it be part of the this PR?

In case people agree at Zulip and these changes are to be included in another PR, I could leave things as they are, or temporarily replace the definition by the local notation `` `ω₁` := (aleph 1).ord ``.

Reply:
Yeah, you should do the Zulip post. These changes should be part of another PR though.

Reply:
Since my last ([somewhat accepted](https://leanprover.zulipchat.com/#narrow/stream/116395-maths/topic/Defining.20initial.20ordinals/near/338425902)) suggestion involved only a few changes, I added it to the PR.